fcrepo-utils/src/main/java/edu/unc/lib/boxc/fcrepo/utils/FedoraSparqlUpdateService.java

```diff
@@ -32,7 +32,7 @@ public void executeUpdate(String uri, String updateString) {
         URI rescUri = URI.create(uri);
 
         try (InputStream sparqlStream = new ByteArrayInputStream(updateString.getBytes(UTF_8))) {
-            try (FcrepoResponse response = fcrepoClient.patch(rescUri)
+            try (FcrepoResponse ignored = fcrepoClient.patch(rescUri)
```

Comment:
intellij did that so that it'd stop warning about the variable being unused. A variable has to be declared in a try with resources block so it can be closed at the end, but it's never used in the block.